kahuna/public/js/services/api/loader.js

```diff
@@ -1,8 +1,11 @@
-import apiServices from '../api';
+import angular from 'angular';
+import {mediaApi} from './media-api';
 
-apiServices.factory('loaderApi',
-                    ['mediaApi',
-                     function(mediaApi) {
+export var loaderApi = angular.module('kahuna.services.api.loader', [
```

Comment:
I thought about that but stuck to `var` for consistency with the rest of the codebase. We can do a general replace throughout as a separate step.

I'm not sure what it means for an export to be `var` or `const`, and specifically what happens to the exported value if you reassign it later..